README.md

```diff
@@ -41,6 +41,13 @@ tweaked to pass the parser.
 If you review the history of the `fec_docs` repo, you'll see some of the types
 of changes that need to be made.
 
+## Concepts
+
+- **Diff**: a structure representing the changes between two regulation trees, describing which nodes were modified, deleted, or added.
```

Comment:
They are conceptually very similar to git diffs, though a bit different in focus (for example, the differences in words rather than lines)